test/suites/scale/deprovisioning_test.go

```diff
@@ -0,0 +1,630 @@
+				provisionerMap[emptinessValue].Spec.TTLSecondsAfterEmpty = lo.ToPtr[int64](0)
+				provisionerMap[expirationValue].Spec.TTLSecondsUntilExpired = lo.ToPtr[int64](0)
+				provisionerMap[expirationValue].Spec.Limits = disableProvisioningLimits
+				for _, p := range provisionerMap {
+					env.ExpectCreatedOrUpdated(p)
+				}
+				driftNodeTemplate.Spec.AMIFamily = &v1alpha1.AMIFamilyBottlerocket
+				env.ExpectUpdated(driftNodeTemplate)
+
+				By("waiting for the nodes across all deprovisioners to get deleted")
+				type testAssertions struct {
+					deletedCount             int
+					deletedNodeCountSelector labels.Selector
+					nodeCount                int
+					nodeCountSelector        labels.Selector
+				}
+				assertionMap := map[string]testAssertions{
+					consolidationValue: {
+						deletedCount: int(float64(nodeCountPerProvisioner) * 0.8),
+						nodeCount:    int(float64(nodeCountPerProvisioner) * 0.2),
+					},
+					emptinessValue: {
+						deletedCount: nodeCountPerProvisioner,
+						nodeCount:    0,
+					},
+					expirationValue: {
+						deletedCount: nodeCountPerProvisioner,
+						nodeCount:    nodeCountPerProvisioner,
+						nodeCountSelector: labels.SelectorFromSet(map[string]string{
+							v1alpha5.ProvisionerNameLabelKey: provisionerMap[noExpirationValue].Name,
+						}),
+					},
+					driftValue: {
+						deletedCount: nodeCountPerProvisioner,
+						nodeCount:    nodeCountPerProvisioner,
+					},
+				}
+				wg = sync.WaitGroup{}
+				for k, v := range assertionMap {
+					wg.Add(1)
+					go func(d string, assertions testAssertions) {
+						defer GinkgoRecover()
+						defer wg.Done()
+
+						env.MeasureDurationFor(func() {
+							// Provide a default selector based on the original provisioner name if one isn't specified
+							selector = assertions.deletedNodeCountSelector
+							if selector == nil {
+								selector = labels.SelectorFromSet(map[string]string{v1alpha5.ProvisionerNameLabelKey: provisionerMap[d].Name})
+							}
+							env.EventuallyExpectDeletedNodeCountWithSelector("==", assertions.deletedCount, selector)
+
+							// Provide a default selector based on the original provisioner name if one isn't specified
+							selector = assertions.nodeCountSelector
+							if selector == nil {
+								selector = labels.SelectorFromSet(map[string]string{v1alpha5.ProvisionerNameLabelKey: provisionerMap[d].Name})
+							}
+							env.EventuallyExpectNodeCountWithSelector("==", assertions.nodeCount, selector)
+							env.EventuallyExpectHealthyPodCount(labels.SelectorFromSet(deploymentMap[d].Spec.Selector.MatchLabels), int(lo.FromPtr(deploymentMap[d].Spec.Replicas)))
+						}, aws.DeprovisioningEventType, multipleDeprovisionersTestGroup, defaultTestName, map[string]string{aws.TestSubEventTypeDimension: d})
+					}(k, v)
+				}
+				wg.Wait()
+			}, aws.DeprovisioningEventType, multipleDeprovisionersTestGroup, defaultTestName)
+		}, SpecTimeout(30*time.Minute))
```

Comment:
Forget what we decided here. Did we say 30 minutes was enough? At 200 nodes each I feel like this will take more than half an hour, but less than an hour, but that's just a guess. 

Edit: looking at the later tests, it says 20 nodes takes 45 minutes for consolidation and drift.So this makes me think this needs to be bumped up.

Reply:
Good callout. You're right that we should bump this